sw/device/lib/testing/edn_testutils.c

```diff
@@ -0,0 +1,60 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/edn_testutils.h"
+
+#include "sw/device/lib/dif/dif_csrng_shared.h"
+#include "sw/device/lib/dif/dif_edn.h"
+#include "sw/device/lib/testing/rand_testutils.h"
+
+/**
+ * Returns randomized seed material.
+ */
+dif_edn_seed_material_t edn_testutils_seed_material_build(bool zero_clen) {
+  dif_edn_seed_material_t seed;
+  seed.len = zero_clen ? 0
+                       : rand_testutils_gen32_range(
+                             0, kDifEntropySeedMaterialMaxWordLen);
+  for (size_t i = 0; i < seed.len; ++i) {
+    seed.data[i] = rand_testutils_gen32();
+  }
+  return seed;
+}
+
+/**
+ * Returns a randomized EDN auto mode configuration.
+ */
+dif_edn_auto_params_t edn_testutils_auto_params_build(bool zero_clen) {
```

Comment:
Part of this function is not the same as it was before (i.e., it's not just a cut-and-paste with some renames). Could you please the functional changes into a separate commit, so this becomes transparent?